### ISSUE TITLE
Can't install Perl on macOS 13.3 M2

### ISSUE THREAD

**Issue by bwva** [gugod/App-perlbrew#774]
On my M2 Mac running macOS 13.3, which comes with perl 5.30.3 and 5.34.0, I am unable to install my own perls. I have tried a direct download install as I have done several times on my old x86 machine, also running macOS 13.3. 

So I tried installing Perlbrew (worked), but it fails to install both 5.36.0 and 5.34.1. I tried using perlbrew again with --force install. No joy. In all install methods, it fails on the same ONE test, t/libperl.t

I tried separately with perlbrew --notest install, and of course it installed (5.36.0). But I am apprehensive about relying on this for my workspace.

I do not know enough about libperl to know how significant this test failure is, but the name suggests to me that it might be more important than say, lib[some_obscure_graphics_format].

Thanks

**Comment by gugod** [comment]
I saw that this seems to have been answered in this reddit thread: https://www.reddit.com/r/perl/comments/12n981g/cant_install_perl_on_macos_133_m2/

tldr: macOS changes breaks its compatibilty `t/libperl.t` and perl has adapted it since perl 5.37.x. For now  test failures from`t/libperl.t` can be ignored.

5.36.1-RC3 is out, perhaps it is worthwhile to try on that version.

**Comment by bwva** [comment]
Yes - fine to close.

    — Bruce

_bruce__van_allen__santa_cruz_ca_

**Comment by gugod** [comment]
FTR this is reproducible also on an intel mac, with macOS Ventura 13.3.1.

```
# perlbrew install -v perl-5.36.1-RC3
......

Test Summary Report
-------------------
porting/libperl.t                                                  (Wstat: 7424 (exited 29) Tests: 0 Failed: 0)
  Non-zero exit status: 29
  Parse errors: No plan found in TAP output
Files=2767, Tests=1206730, 250 wallclock secs (70.80 usr 14.90 sys + 508.88 cusr 102.38 csys = 696.96 CPU)
Result: FAIL
```

I guess we would need some updates in patchperl to backport the changes so it at least would finish correctly on perlbrew.

**Comment by bwva** [comment]
> I guess we would need some updates in patchperl to backport the changes so it at least would finish correctly on perlbrew.


Yes, it was quite surprising that it wouldn’t install with perlbrew.

If you haven’t looked, that test, porting/libperl.t, already has some comments about macOS complications, so someone might already be familiar with how to track down what the test should allow under macOS 13.3+.

Thanks,

    — Bruce

_bruce__van_allen__santa_cruz_ca_
